File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 4/src/Ex2.java
//Importando Scanner
import java.util.Scanner;

public class Ex2 {

	public static void main(String[] args){
		
		//Instanciando Scanner
		Scanner input = new Scanner(System.in);
		
		System.out.println("Vetor dos 10 primeiros ímpares\n\n");
		
		//vars
		int impares[] = new int[10];
		int i=0,  num=1;
		
		//Entrada 
		while( i < 10)
		{
			//Processamento
			if( num%2 == 1)
			{
				impares[i] = num;
				i++;
			}
			num++;
		}
		
		//Saída
		System.out.println("Vetor:");
		for(i=0; i<10; i++)
		{
			System.out.print(impares[i] + "  ");
		}
		
		
		
	}
}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 4/ex2.c
#include <stdio.h>

main()
{
    printf("10 Primeiros Impares\n\n");

    //vars
    int v[10], i=1, cont=0;

    //Entrada e Processamento
    while( cont < 10)
    {
        if( i%2 != 0 )
        {
            v[cont] = i;
            cont++;
        }
        i++;
    }

    //Saída
    printf("\nVetor Resultante: \n");
    for( i=0; i<10; i++)
    {
        printf("%d ", v[i]);
    }
}
